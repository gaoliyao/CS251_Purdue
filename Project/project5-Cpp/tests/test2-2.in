2
100 600
AAA ABV 188.46
AAB ABS 533.61
AAC ABG 543.06
AAD ABA 799.92
AAE ADO 320.66
AAF ADL 80.45
AAG AAQ 814.90
AAH AAV 706.35
AAI AAC 144.66
AAJ ABC 725.55
AAK ACV 877.04
AAL ACR 960.72
AAM ABS 131.30
AAN ABH 550.33
AAO ACM 99.14
AAP AAO 180.63
AAQ ACJ 127.10
AAR ACZ 819.45
AAS AAQ 376.55
AAT ADL 903.09
AAU AAV 296.78
AAV AAG 287.89
AAW AAU 305.27
AAX AAY 230.74
AAY ABT 751.05
AAZ ABD 314.19
ABA AAJ 164.98
ABB ABR 233.53
ABC ADC 812.76
ABD ABN 234.97
ABE ABQ 105.05
ABF AAM 526.55
ABG AAP 456.89
ABH AAA 902.31
ABI ABT 292.42
ABJ ADO 694.18
ABK ACU 115.99
ABL AAI 191.45
ABM ADA 139.84
ABN AAT 198.24
ABO ADL 378.49
ABP ADR 682.51
ABQ ADV 114.38
ABR ACE 330.32
ABS ACO 26.95
ABT ABD 166.08
ABU AAA 144.41
ABV ACM 364.15
ABW ADF 202.16
ABX ADV 761.80
ABY AAK 398.91
ABZ ABY 410.83
ACA ADM 765.10
ACB ADC 263.21
ACC ADR 187.99
ACD ABT 959.36
ACE AAA 300.92
ACF ABN 437.00
ACG ABE 884.96
ACH ABG 505.96
ACI ABR 496.13
ACJ ACE 370.96
ACK AAG 528.20
ACL ADL 730.37
ACM ACK 494.21
ACN ABV 727.54
ACO ACU 962.82
ACP ADS 878.82
ACQ ACK 669.17
ACR ABD 916.62
ACS AAO 70.18
ACT AAO 595.06
ACU ACF 247.14
ACV ADK 924.05
ACW ADU 843.36
ACX ABX 837.63
ACY ADP 370.45
ACZ ACS 207.13
ADA AAI 374.11
ADB AAC 761.23
ADC ABH 653.02
ADD ACZ 145.57
ADE ABK 58.25
ADF ABN 507.99
ADG ADR 135.21
ADH AAN 185.09
ADI ABN 398.27
ADJ AAQ 80.96
ADK ADQ 954.04
ADL ACC 150.44
ADM ABP 168.03
ADN AAI 639.86
ADO AAA 629.78
ADP ADV 134.60
ADQ ADL 111.54
ADR ADF 35.74
ADS ACE 548.12
ADT ABF 543.92
ADU ADM 205.71
ADV ABR 660.85
ADO AAI 845.94
ADF ADK 789.62
ABF ADE 907.19
ADH ADA 868.12
ABO ADI 507.98
ADO ACM 354.97
ACB AAG 466.51
ADK ADC 394.59
ACB AAM 102.04
ADR ABJ 317.47
ADC ADB 284.56
ABA ACK 813.44
ADL ADP 314.78
ACI ABF 966.56
ACU ACT 606.43
ACS AAQ 39.59
AAT ACR 640.17
AAX AAJ 112.98
ADE AAO 930.62
ADI AAJ 953.97
ADI ABP 645.28
ABQ ACQ 46.42
ABM AAJ 401.87
ACQ ACR 880.18
ACY ABP 976.42
ACT AAL 392.58
ABF ABE 426.79
AAA AAF 974.88
ABY ADK 75.55
AAM ABA 171.64
ACJ AAM 251.06
ABP AAG 323.26
ADF ADS 14.35
ACT ACM 395.04
ADL ABP 333.45
ABF ACM 501.08
ADM AAU 596.48
ADS ACJ 547.98
ADG AAQ 593.63
ACK ABG 593.24
ABB ADM 401.66
ABH AAQ 256.49
ADE AAD 513.20
ADU AAH 813.62
ADN ACA 90.45
ACE AAM 759.87
AAH ABZ 425.99
ACQ ABY 383.35
ABK ACN 789.97
AAV ADL 1.24
AAL AAH 657.10
ADC AAJ 210.92
ADF ADD 757.90
ABR ADR 359.82
ABV ABP 848.38
ACI ABC 258.41
ACL ADJ 428.45
ACK AAI 982.95
ABX ABS 294.60
ABU ACN 363.01
AAX ABD 996.61
ABH ACJ 50.22
ACZ ADS 780.55
ABZ AAB 823.87
ABS ADN 891.47
ACU ADH 329.62
AAL ABE 193.76
AAT ACZ 441.25
ACM ADL 642.23
ADD ABQ 7.98
ACK AAD 30.83
AAX ACZ 264.60
ACU ABH 419.80
AAL ABK 98.33
ABY ADC 625.25
ADT ACA 737.62
AAL AAP 114.92
ADJ ABJ 638.69
ACM ABZ 620.58
ADK ABG 292.52
ADI ABV 797.56
AAW ACS 679.85
ADE ABQ 179.18
AAO AAG 330.06
ABW ACE 439.86
AAZ AAG 229.90
ADJ AAR 114.06
ADB ACF 760.41
AAA AAV 564.92
AAK ACJ 296.76
ACK AAA 158.23
ABU ACE 737.04
ABO ABA 162.71
ABU ABH 686.71
ACD AAT 835.96
ACJ ACK 886.35
ABE ACQ 888.14
ADJ AAJ 855.66
AAJ ACB 443.76
ACS ABD 487.90
AAW ACB 329.61
ABS ABZ 14.69
AAB ADN 42.47
ACU ADM 236.80
ABR ABT 34.52
ADN AAG 62.14
ABA ABK 203.82
ADO AAY 269.43
ACG ADH 899.96
ADI AAH 990.77
ABC ABE 239.82
ABP ACW 147.33
AAK ABB 902.25
ACG ADV 256.15
ADM ADQ 183.60
ADQ ADH 264.18
AAI ACL 685.06
ABQ ACD 761.82
ADH ACN 379.67
ABH AAD 126.27
AAL ABG 318.57
ADP ACV 341.83
ADB ABK 933.11
AAE ABU 418.62
ACT ADK 485.09
ADQ AAB 297.43
ACG ACH 344.86
ACN ABQ 825.23
ABF ACJ 369.09
ABI ABO 740.20
AAS ADT 514.65
ABS ABH 599.96
ADC ADS 424.00
ADA ACG 831.23
ACO AAT 422.06
AAB AAO 440.59
AAT ABF 742.89
AAN ACC 318.72
ABA ACL 391.59
ACP ABJ 430.77
AAQ ACB 562.21
AAB ABX 926.53
ADT AAC 543.31
AAI AAZ 159.50
ABB ADG 229.51
ADP ABR 596.96
AAR AAO 472.79
ACH ADB 10.34
ABF AAG 80.49
ADM AAZ 54.37
AAU ABP 706.42
ABP ABR 958.43
ABV ABO 634.45
ABE AAA 305.23
AAH ADG 974.21
AAC ADC 854.91
AAR ACI 642.02
ABW ABU 584.33
ACC ABV 81.76
ADS ACV 251.78
ADL AAQ 210.22
ABK AAI 8.19
ACP AAJ 476.94
ADV ADL 875.01
ADF ACK 133.56
ACV ABC 222.07
ACV ABA 806.41
ACR ACH 938.62
AAR ABG 190.40
ACG AAA 564.14
ABV AAJ 735.86
ACO AAS 212.80
ACN ACH 251.34
AAB AAV 384.90
AAR ABB 618.15
ADT ABI 728.88
AAX ABB 440.16
ABC AAK 519.65
ACI AAD 735.76
ACJ AAR 196.23
AAW ACD 515.41
ADO ABK 813.06
ABZ AAW 419.59
ACU AAM 444.76
ABZ ABB 452.25
ADI AAP 198.48
ACN ADG 337.97
AAH ABF 258.76
AAK AAH 616.89
ADC ADE 976.11
ABJ AAX 657.03
AAX ABE 247.06
ABT ABP 144.31
AAJ ACS 928.82
ABK ADR 986.20
AAE ABS 122.86
ADK ABP 554.00
ACA AAA 668.31
ADC AAL 518.39
ACS ABR 891.73
ADD ABE 616.97
ABL AAG 318.79
ABR ABZ 440.20
ACW AAN 169.90
AAJ ABX 351.86
ABD AAF 732.30
AAP ADP 614.23
AAP ADT 912.23
ADV ADA 58.34
ADM AAA 539.29
ACL ADD 923.60
ACI AAS 646.66
ABN AAO 231.69
ACI ADK 586.82
ABY ADT 401.84
ADL AAS 489.94
ABJ ABP 441.62
ACJ AAO 735.51
AAD AAO 274.81
ACO ABW 361.93
ABB ACO 8.59
AAS ABG 240.28
ADA AAV 990.63
ADD ACS 392.47
ACR ABX 487.88
ADK ADS 687.23
ADD AAK 123.08
ADU ACT 186.74
AAC ACW 395.89
ABL ABC 779.19
ACD ADJ 766.90
ADK ACP 160.68
ACJ ADE 621.01
ACT ADH 290.70
AAL ABH 977.93
ADO ABR 264.53
ABR ACO 614.80
ABT ADO 10.69
ADC ADG 72.58
ACI ACE 571.65
ABV ABU 331.05
AAS ABX 518.68
AAC ABE 854.91
ABN ACA 730.02
AAY ADU 72.46
ACP ABZ 559.41
ABH ABF 762.86
ACZ ACA 448.18
ADA ACR 980.80
ACZ AAP 311.85
ADS ABI 682.04
ABT AAC 515.58
ABB ABT 492.49
ABG ABV 782.73
ABE ABT 161.40
ABH ADJ 792.25
AAR AAZ 39.02
ACR ABS 313.15
ADG ACW 393.85
ACM ACA 211.52
ABX ABW 926.86
AAF ABT 324.40
AAP AAY 576.85
ACM ACE 722.84
ABL ADI 104.82
ABD ACU 46.21
AAG ADL 750.46
AAI AAK 66.42
AAY ADQ 196.17
AAV AAN 351.12
ACR ACA 812.17
ACO ADO 952.33
ADM ABE 748.49
ADV AAO 327.39
ABL ABW 67.68
AAJ ACR 763.75
ADU AAQ 662.35
AAJ AAK 874.11
ACD ADB 479.72
ACO AAA 460.38
ADM ADR 469.87
ABI ADC 361.78
ACM ABE 791.29
ADB AAV 952.66
ADB ADL 914.76
ABX AAQ 581.74
AAR ACA 194.35
ABB AAP 911.91
AAH ADK 201.57
ACH ABA 842.35
ADD ACQ 458.82
ACQ AAC 27.63
ADU ACP 508.45
ABB ADU 316.66
ADQ ACZ 356.07
ACS ADA 322.76
ADS ADR 951.80
ABG AAX 454.95
AAX ADE 437.83
ADC ABP 44.61
ACP AAW 182.81
ABX ABR 920.84
ACM ABP 558.03
ACJ ACR 626.01
ADB ACL 415.30
ABN ADH 772.09
AAP ADD 620.04
ADR AAN 112.79
ADM ADK 600.13
ABA AAU 953.07
ABL AAY 766.05
ACB ACO 324.08
AAP ADJ 113.61
ADQ ABY 528.91
ACA ABW 593.20
AAJ ADR 620.86
ADK ADI 242.74
AAC AAM 823.95
AAJ ABZ 206.82
ACT ACP 697.77
AAZ ABH 185.37
ADO ACU 462.50
ADI ACW 991.42
ACU ACJ 619.46
ACU ADD 867.55
ACQ ACP 391.96
AAG ABQ 516.50
AAX AAB 651.94
ABV AAU 513.23
ACU ACB 698.61
ACM AAZ 161.11
AAC ADV 316.05
AAE ADV 650.94
ACH AAT 944.13
ACD AAZ 480.07
AAF ABR 839.96
AAQ ACM 733.72
ACS ACG 641.19
ADM ADE 984.43
ACH ACV 665.58
ACJ AAZ 981.64
ABO ABL 244.20
ADB ACG 764.49
ADJ ADF 244.56
ADO ADA 248.05
ACJ ABQ 981.77
ABF ACB 786.48
ACV ADJ 770.91
AAV ABI 80.02
ADR ADO 354.15
ACU AAI 833.55
ACJ ABC 313.94
ABT AAI 35.04
ABU ADR 715.87
AAC ABN 317.17
AAH AAA 68.60
ABK ADF 112.65
ABK AAJ 635.52
ADB ABE 662.86
ADD AAZ 519.30
ACH ACP 683.99
ACE ABB 719.03
ACO ACI 624.16
AAS ACU 842.04
AAC ADL 846.41
ACL ADP 948.72
ADP ABF 967.54
ACT ACK 504.03
AAU ACB 874.07
ABF AAU 721.58
ADJ AAA 604.14
ACF ADU 721.40
ACR ACO 969.59
ADQ ACS 359.13
ABJ ACL 299.76
ABD ABK 443.21
ACI ADM 587.92
ACW ABT 723.13
ABB ACA 680.44
AAA AAL 111.84
AAA AAZ 270.25
ABA ABI 143.75
ACP ACD 641.41
ADA AAB 144.64
ABR ABL 494.27
ACT ACO 792.63
AAF ADH 85.53
ABY ADU 359.80
ACD ADQ 326.80
ACO ADA 882.80
AAH ABD 522.31
ABA ADG 327.25
ACE ADI 635.41
ADV AAX 129.68
AAW ABP 85.84
AAO AAT 307.89
ADV ACU 626.92
ABX ACB 753.47
ADS ABH 572.65
ACW ACO 688.60
ADE AAZ 303.89
ACV ADN 438.59
AAK ABI 659.52
ABG ACX 745.36
ADU ADR 216.78
ACS ADV 188.72
ACR ABV 615.58
ABI AAW 88.76
ADQ AAE 839.01
ADF ACS 472.56
AAI AAN 911.15
AAU ADT 416.77
AAG AAC 60.61
ABI ABM 520.59
ABD ACQ 226.09
ABW ABM 166.09
ACV AAH 664.08
ABE AAH 291.39
ADJ ACI 780.74
AAJ AAQ 133.83
ACX ADF 938.55
AAY AAM 173.61
ADR AAR 951.32
ACY ADH 177.42
ACZ ACY 507.03
ABY ABJ 171.11
ADD ABD 508.94
ABO ACQ 238.33
ACA ADR 762.94
AAU ADA 813.69
ACD ADH 247.46
ACP ABU 396.72
ABO ADQ 497.41
ACU ACS 602.79
AAO ABV 556.55
ACR ADT 282.90
ACT ACX 701.22
ABA ADJ 808.54
ABU AAR 622.24
AAB AAC 33.22
AAU AAA 429.94
ACI ADQ 90.88
ADH AAR 693.67
ADV ACM 250.22
AAV AAO 696.64
AAC ADS 975.36
ABD ACT 704.80
ADL ADT 338.67
ABR ADS 501.36
AAT ABX 804.38
ABP ACS 748.38
ADK ABE 89.85
ACY AAP 353.02
ABR ACP 736.18
AAR ADK 646.32
ADD ACR 861.71
ACA ABX 780.60
ABX ACQ 671.55
ADT AAE 184.75
ABV ABQ 731.16
AAR ABH 871.04
ABV AAS 265.32
AAP ACU 189.75
ADO AAW 750.75
ABA ACM 46.56
ADR ADJ 693.08
ABT ADV 250.09
ACH AAS 379.29
AAG AAM 273.98
AAX ADT 308.54
AAW AAP 737.39
ADL ADJ 927.14
ACH ABT 401.05
ADH ACJ 447.61
ABG ABW 304.21
ACZ ABV 717.82
ABR ACC 405.41
ACJ ABJ 580.99
AAZ ADH 198.74
AAX ABO 771.49
ADM ADB 159.38
ACK ACI 835.56
ACH AAR 343.93
ACV ADA 752.26
ACV ABZ 939.03
ACV ABH 448.46
ACH ACK 470.26
AAB AAM 599.40
ABY AAC 725.71
ABO ADG 779.16
ACC ADS 778.25
AAO AAN 122.18
ABN ABR 874.44
AAM ADQ 977.00
ADH ADC 588.98
ACM ACH 915.10
ABB ADH 846.47
ABB ABM 199.59
AAT ABO 479.13
ADM ADD 394.50
ACG ABJ
ABU ADF
AAB AAM
AAC ADO
ACD ABU
AAF AAC
ABU ABQ
ABI ABT
AAH ADP
ACD ADI
END
