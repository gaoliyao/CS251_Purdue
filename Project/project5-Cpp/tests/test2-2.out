ACG ADV ABQ ABE ADK ABY ABJ 812.09
ABU AAA ACK ADF 436.20
AAB AAC AAG AAM 367.81
AAC AAI ABT ADO 190.39
ACD ADH AAZ AAA ABU 860.86
AAF ADL AAV AAG AAC 430.19
ABU AAA ABE ABQ 554.69
ABI ABT 292.42
AAH ABE ABQ ADV ADP 645.42
ACD ADH AAZ AAI ABL ADI 901.97
